x-pack/platform/plugins/shared/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -167,14 +178,20 @@ export const CaseViewActivity = ({
 
   const handleUserActionsActivityChanged = useCallback(
     (params: UserActivityParams) => {
-      setUserActivityQueryParams((oldParams) => ({
-        ...oldParams,
-        page: 1,
-        type: params.type,
-        sortOrder: params.sortOrder,
-      }));
+      setUserActivityQueryParams((oldParams) => {
+        if (oldParams.sortOrder !== params.sortOrder) {
+          setSortOrder(params.sortOrder);
+        }
```

Comment:
I think is better if we move the call outside `setUserActivityQueryParams` (even if we cannot check the old params) to allow React do its magic and chain the two state calls. I am not sure if there are any side effects with setting a state inside another setter.